### PR TITLE
guides/sev: point container image link to web UI

### PR DESCRIPTION
**Pull request by katexochen**
Linking to the container registry directly isn't that helpful when reading the docs.

### CODE REVIEW

**Review by wainersm — APPROVED** [review]

File: guides/sev.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Setting the second bit of the policy enables SEV-ES.
 For more information see chapter 3 of the Secure Encrypted Virtualization API.
 The SEV policy is not the same as the policies that drive `simple-kbs`.
 
-The CoCo project has created a sample encrypted container image ([encrypted-image-tests](ghcr.io/fitzthum/encrypted-image-tests:encrypted)). This image is encrypted using a key that comes already provisioned inside the `simple-kbs` for ease of testing. No `simple-kbs` policy is required to get things running.
+The CoCo project has created a sample encrypted container image ([ghcr.io/fitzthum/encrypted-image-tests:encrypted](https://github.com/users/fitzthum/packages/container/encrypted-image-tests/61923658?tag=encrypted)). This image is encrypted using a key that comes already provisioned inside the `simple-kbs` for ease of testing. No `simple-kbs` policy is required to get things running.
```
Comment (fitzthum) [inline guides/sev.md:77]: I think we may want to switch this to https://github.com/orgs/confidential-containers/packages/container/test-container/82546195?tag=unencrypted

The image in my GHCR is probably from before v0.6.0 and might not work. Interestingly that image was pulled 300+ times today so I guess someone is still using it somewhere.

Comment (katexochen) [reply guides/sev.md]: Updated it. Do you know where the Dockerfile for that container lives? I would love to establish a container naming pattern like `ghcr.io/<org>/<repo>/<container-name>`.

**Review by fitzthum — APPROVED** [review]

File: guides/sev.md (hunk 1 above)
Comment (fitzthum) [reply guides/sev.md]: I'm not sure the Dockerfile is tracked anywhere publicly @ryansavino?

Comment (ryansavino) [reply guides/sev.md]: I don't think it's in source control currently, but this section has the Dockerfile contents:
https://github.com/confidential-containers/confidential-containers/blob/main/guides/sev.md#create-an-encrypted-image

Comment (fitzthum) [reply guides/sev.md]: Yeah although that doesn't do the label stuff.

**Pull request merged**